bees.schema.json

```diff
@@ -14,21 +14,27 @@
         "type": "object",
         "properties": {
           "name": {
+            "description": "This flower's common name.",
             "type": "string",
             "not": {
-              "pattern": "\\bor\\b"
+              "pattern": "\\bor\\b",
+              "description": "If there are two common names, pick one, rather than joining both together with the word 'or'."
             }
           },
           "sci_name": {
+            "description": "This flower's scientific name.",
             "type": "string",
-            "pattern": "^[A-Z][a-zäëïöüÿ\\-]* [a-z][a-zäëïöüÿ\\-]+$"
+            "pattern": "^[A-Z][a-zäëïöüÿ\\-]* [a-z][a-zäëïöüÿ\\-]+$",
+            "$comment": "See https://www.iapt-taxon.org/nomen/pages/main/art_60.html for the precise structure of scientific names."
           },
           "relative_size": {
+            "description": "Roughly how large this bee is compared to an average bee. 1 is normal, 0.1 is small, and 10 is big.",
```

Comment:
I think in practice the range of reasonable sizes is more like 0.1-2 and even that is probably more range than is actually used. We probably want to bring down the maximum here.